### ISSUE TITLE
Add Casual Agenda

### ISSUE THREAD

**Issue by kickingvegas** [kickingvegas/casual-suite#34]
Add https://github.com/kickingvegas/casual-agenda

**Comment by kickingvegas** [comment]
Blocked by https://github.com/kickingvegas/casual-agenda/issues/7

**Comment by kickingvegas** [comment]
Merged into development
